pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -27,6 +27,49 @@ import (
 
 var chartPathControlPlane = filepath.Join(common.ChartPath, "seed-controlplane", "charts")
 
+// GetResources returns the cpu and memory requirements for API server based on nodeCount
+func (b *HybridBotanist) GetResources(nodeCount int) (string, string, string, string) {
```

Comment:
I would suggest to provide a better function name here and to make the function private.

Reply:
Why is the function a method on the `HybridBotanist` object? It's not needed.